docs/src/tutorials/bseries_basics.md

```diff
@@ -0,0 +1,314 @@
+# [B-Series basics](@id tutorial-bseries-basics)
+In this tutorial we use `bseries.jl` to investigate error expansions for RK
+methods, generically or when applied to a specific ODE.
+
+
+```julia
```

Comment:
```suggestion
```@example ex:bseries-basics
```
This allows us to continue the example code below and let Documenter.jl run the code when creating the docs.

Reply:
I think that this and your other comments below were made at the same time that I was already making the changes you had in mind.  Can you take a look at the current version?

Reply:
Yes, you're right. The current version is nice :+1: